### PR TITLE
Fix: Adding default vault to quota parameter (PROJQUAY-0000)

### PR DESCRIPTION
**Pull request by Sunandadadi**

### CODE REVIEW

File: util/ipresolver/aws-ip-ranges.json [hunk 1]
```diff
@@ -46895,4 +46895,4 @@
       "network_border_group": "us-west-2"
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (kwestpharedhat) [inline util/ipresolver/aws-ip-ranges.json:46898]: Can we revert this file?

File: static/partials/org-view.html [hunk 2]
```diff
@@ -140,7 +140,9 @@ <h3>Organization Settings</h3>
 
               <time-machine-settings organization="organization"></time-machine-settings>
 
-              <quota-management-view quay-show="Features.QUOTA_MANAGEMENT" organization="organization" disabled="true"></quota-management-view>
+              <div quay-show="Features.QUOTA_MANAGEMENT">
```
Comment (kwestpharedhat) [inline static/partials/org-view.html:143]: Is there a reason to switch to a div rather than the show parameter being inside the view object?

**Review by kwestpharedhat — DISMISSED** [review]

File: util/ipresolver/aws-ip-ranges.json (hunk 1 above)
Comment (Sunandadadi) [reply util/ipresolver/aws-ip-ranges.json]: Yes, done.

**Review by kwestpharedhat — APPROVED** [review]

**Pull request merged**